phalcon/Acl/Adapter/Memory.zep

```diff
@@ -507,6 +507,34 @@ class Memory extends AbstractAdapter
         }
      }
 
+    /**
+     * Return an array with every component registered in the list
+     */
+    public function getComponents() -> <ComponentInterface[]>
+    {
+        return this->components;
+    }
+
+    /**
+     * Returns the inherited roles for a passed role name. If no role name
+     * has been specified it will return the whole array. If the role has not
+     * been found it returns an empty array
+     */
+    public function getInheritedRoles(string roleName = "") -> array
```

Comment:
```suggestion
    public function getInheritedRoles(string roleName = null) -> array
```

I think it make more sense to this be nullable string. As role, in some extreme cases, might be empty string.

Reply:
Discussed and left it as is.